crates/astria-sequencer/src/mempool/transactions_container.rs

```diff
@@ -129,28 +134,35 @@ impl fmt::Display for TimemarkedTransaction {
 struct TransactionPriority {
     nonce_diff: u32,
     time_first_seen: Instant,
+    group: Group,
 }
 
 impl PartialEq for TransactionPriority {
```

Comment:
Isn't this the same as deriving `PartialEq`?

Reply:
Can you say more about what you mean by this? I'm a little confused

Reply:
I meant that your manual implementation looks the same as deriving `PartialEq`.